rewrite-core/src/main/java/org/openrewrite/config/Environment.java

```diff
@@ -150,28 +150,27 @@ public Recipe activateRecipes(Iterable<String> activeRecipes) {
             }
         }
         if (!recipesNotFound.isEmpty()) {
+            @SuppressWarnings("deprecation")
             List<String> suggestions = recipesNotFound.stream()
                     .map(r -> recipesByName.keySet().stream()
                             .min(comparingInt(a -> StringUtils.getLevenshteinDistance(a, r)))
                             .orElse(r))
                     .collect(toList());
-            String message = String.format("Recipes not found: %s\nDid you mean: %s",
+            String message = String.format("Recipe(s) not found: %s\nDid you mean: %s",
                     String.join(", ", recipesNotFound),
                     String.join(", ", suggestions));
             throw new RecipeException(message);
         }
+        if (activatedRecipes.size() == 1) {
+            return activatedRecipes.get(0);
+        }
         return new CompositeRecipe(activatedRecipes);
```

Comment:
We could return `Recipe.noop()` here if the actived recipes is empty, because an empty array was passed into `activateRecipes(String... activeRecipes)` below.

Reply:
That sounds better to me as well.